src/lib/ecore_con/Ecore_Con.h

```diff
@@ -96,38 +101,38 @@
 /**
  * @defgroup Ecore_Con_Buffer Ecore Connection Buffering
  * @ingroup Ecore_Con_Group
- * 
+ *
  * As Ecore_Con works on an event driven design, as data arrives, events will
  * be produced containing the data that arrived. It is up to the user of
  * Ecore_Con to either parse as they go, append to a file to later parse the
  * whole file in one go, or append to memory to parse or handle later.
- * 
- * To help with this Eina has some handy API's. The Eina_Binbuf and 
- * Eina_Strbuf APIs, abstract dynamic buffer management and make it trivial 
- * to handle buffers at runtime, without having to manage them. Eina_Binbuf 
- * makes it possible to create, expand, reset and slice a blob of memory - 
- * all via API. No system calls, no pointer manipulations and no size 
+ *
+ * To help with this Eina has some handy API's. The Eina_Binbuf and
+ * Eina_Strbuf APIs, abstract dynamic buffer management and make it trivial
+ * to handle buffers at runtime, without having to manage them. Eina_Binbuf
+ * makes it possible to create, expand, reset and slice a blob of memory -
+ * all via API. No system calls, no pointer manipulations and no size
```

Comment:
oh boy... @JPTIZ, are you gonna make me add trailing spaces again?

Reply:
`git checkout -p -- <file>` and use `y/n/s` :^D

JK, maybe we can solve this later.